renovate.json

```diff
@@ -1,121 +1,39 @@
 {
-  "enabled": true,
   "extends": [
-    ":dependencyDashboard",
+    "github>defenseunicorns/uds-common//config/renovate.json5",
     ":semanticCommits",
     ":semanticCommitTypeAll(chore)"
   ],
-  "forkProcessing": "enabled",
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "branchConcurrentLimit": 0,
-  "prConcurrentLimit": 0,
-  "prHourlyLimit": 0,
-  "rebaseStalePrs": true,
-  "rebaseWhen": "conflicted",
-  "commitBodyTable": true,
-  "separateMajorMinor": false,
```

Comment:
Maybe we don't care/need this but it [defaults to true](https://docs.renovatebot.com/configuration-options/#separatemajorminor) and common does not override unless its somewhere in the base config?